src/Components/WebAssembly/WebAssembly/src/Http/WebAssemblyHttpRequestMessageExtensions.cs

```diff
@@ -0,0 +1,169 @@
+            };
+
+            return SetBrowserFetchOption(requestMessage, "mode", stringOption);
+        }
+
+        /// <summary>
+        /// Configures a value for the 'integrity' option for the HTTP request.
+        /// </summary>
+        /// <param name="requestMessage">The <see cref="HttpRequestMessage"/>.</param>
+        /// <param name="integrity">The subresource integrity descriptor.</param>
+        /// <returns>The <see cref="HttpRequestMessage"/>.</returns>
+        /// <remarks>
+        /// See https://developer.mozilla.org/en-US/docs/Web/API/Request/integrity
+        /// </remarks>
+        public static HttpRequestMessage SetBrowserIntegrity(this HttpRequestMessage requestMessage, string integrity)
+            => SetBrowserFetchOption(requestMessage, "integrity", integrity);
+
+        /// <summary>
+        /// Configures a value for the HTTP request.
+        /// </summary>
+        /// <param name="requestMessage">The <see cref="HttpRequestMessage"/>.</param>
+        /// <param name="name">The name of the HTTP fetch option.</param>
+        /// <param name="value">The value, which must be JSON-serializable.</param>
+        /// <returns>The <see cref="HttpRequestMessage"/>.</returns>
+        /// <remarks>
+        /// See https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/fetch
+        /// </remarks>
+        public static HttpRequestMessage SetBrowserFetchOption(this HttpRequestMessage requestMessage, string name, object value)
+        {
+            if (requestMessage is null)
+            {
+                throw new ArgumentNullException(nameof(requestMessage));
+            }
+
+            const string FetchRequestOptionsKey = "WebAssemblyFetchOptions";
+            IDictionary<string, object> fetchOptions;
+
+            if (requestMessage.Properties.TryGetValue(FetchRequestOptionsKey, out var entry))
+            {
+                fetchOptions = (IDictionary<string, object>)entry;
+            }
+            else
+            {
+                fetchOptions = new Dictionary<string, object>();
+                requestMessage.Properties[FetchRequestOptionsKey] = fetchOptions;
+            }
+
+            fetchOptions[name] = value;
+
+            return requestMessage;
+        }
+
+        /// <summary>
+        /// Configures streaming response for the HTTP request.
+        /// </summary>
+        /// <param name="requestMessage">The <see cref="HttpRequestMessage"/>.</param>
+        /// <param name="streamingEnabled"><see langword="true"> if streaming is enabled; otherwise false.</param>
+        /// <returns>The <see cref="HttpRequestMessage"/>.</returns>
+        /// <remarks>
+        /// This API is only effective when the browser HTTP Fetch supports streaming.
+        /// See https://developer.mozilla.org/en-US/docs/Web/API/ReadableStream.
+        /// </remarks>
+        public static HttpRequestMessage EnableStreamingResponse(this HttpRequestMessage requestMessage)
```

Comment:
```suggestion
        public static HttpRequestMessage SetBrowserResponseStreamingEnabled(this HttpRequestMessage requestMessage, bool enabled)
```

Name suggestion is so it appears alphabetically with the others in intellisense.

Suggestion to make it possible to set "false" is because maybe in the future it might default to true, and you need a way to turn it off. Also with all the other methods here, it's possible to undo them by overwriting with `null` or similar, so it's nice if this one is undoable too.

What do you think?